doc/getting-started/getting-started.rst

```diff
@@ -507,23 +529,39 @@ Build ACRN
       .. code-block:: bash
 
          disk="/media/$USER/"$(ls /media/$USER)
-         sudo cp linux-5.10.47-acrn-sos-x86.tar.gz $disk/
-         sudo cp ~/acrn-work/acrn-hypervisor/build/hypervisor/acrn.bin $disk/
-         sudo cp ~/acrn-work/my_board3/output/launch_uos_id3.sh $disk/
-         sudo cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl $disk/
-         sudo cp ~/acrn-work/acrn-hypervisor/build/acrn-2.6-unstable.tar.gz $disk/
+         cp linux-5.10.47-acrn-sos-x86.tar.gz $disk/
+         cp ~/acrn-work/acrn-hypervisor/build/hypervisor/acrn.bin $disk/
+         cp ~/acrn-work/my_board/output/launch_uos_id3.sh $disk/
+         cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl $disk/
+         cp ~/acrn-work/acrn-hypervisor/build/acrn-2.6-unstable.tar.gz $disk/
          sync && sudo umount $disk/
 
-   #. Insert the USB disk you just used into the target system and run these commands:
+   #. Insert the USB disk you just used into the target system and run these
+      commands to copy the tar files locally:
```

Comment:
Recommend formatting the following steps as substeps, rather than using "and" at the beginning of each step.